Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```diff
@@ -681,6 +687,42 @@ script:
     - contextPath: Gra.Highrisk.Users.exitDate
       description: Exit Date
       type: date
+    - contextPath: Gra.Highrisk.Users.updated_on
+      description: Updated On
+      type: date
+    - contextPath: Gra.Highrisk.Users.created_on
+      description: Created On
+      type: date
+    - contextPath: Gra.Highrisk.Users.resource
+      description: Resource
+      type: string
+    - contextPath: Gra.Highrisk.Users.name
+      description: Name
+      type: string
+    - contextPath: Gra.Highrisk.Users.id
+      description: Id
+      type: string
+    - contextPath: Gra.Highrisk.Users.type
+      description: Type
+      type: string
+    - contextPath: Gra.Highrisk.Users.description
+      description: Description
+      type: string
+    - contextPath: Gra.Highrisk.Users.is_reassigned
+      description: Is Reassigned
+      type: string
+    - contextPath: Gra.Highrisk.Users.high_risk
+      description: High Risk
+      type: string
+    - contextPath: Gra.Highrisk.Users.is_orphan
+      description: Is Orphan
+      type: string
+    - contextPath: Gra.Highrisk.Users.domain
+      description: Domain
+      type: string
+    - contextPath: Gra.Highrisk.Users.risk_score
```

Comment:
Is there a difference between this field and the one in [#L660](https://github.com/demisto/content/blob/20e04ef44ae91a10fe80b86ca84a43f14a4ec1d5/Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml#L660)?

Reply:
Removed line